.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+Please fill in the fields below to submit a pull request.  The more information that is provided, the better.
+
+Fixes #Issue:
```

Comment:
For GitHub to automatically close the issue we need to use a specific format see [using keywords in issues and pull requests](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests).
```suggestion
Fixes: #<Issue>
```
?

Reply:
doesn't look like it needs the ":", so removing that